src/app-cluster.service.ts

```
@@ -6,19 +6,19 @@ const numCPUs = os.cpus().length;
-            for (let i = 0; i < numCPUs; i++) {
-                cluster.fork();
-            }
-            cluster.on('exit', (worker, code, signal) => {
-                console.log(`Worker ${worker.process.pid} died. Restarting`);
-                cluster.fork();
-            })
-        } else {
-            console.log(`Cluster server started on ${process.pid}`)
-            callback();
-        }
+  static clusterize(callback: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)